src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -194,8 +207,27 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin, EnumMixin)
       this.saving = false
       return
     }
-
-    const data = {
+    const data = this.isAdministrativeDissolutionOrPutBackOn ? {
```

Comment:
Could you break this up into separate consts please?
You will have to switch between the objects in the code that uses them.

Reply:
will do, are there any specific names i should be using for the consts